### ISSUE TITLE
Implement link option in steam_id on embed

### ISSUE THREAD

**Issue by crashzk** [TheByKotik/sbpp_discord#5]
I believe it would be a good resource if the steam of the players that carry Bans, Comms or even Report have the option to link on Steam of the same, then clicking would open the steam of the same.
![image](https://user-images.githubusercontent.com/32937653/102007760-8bd6a780-3d0a-11eb-8cfd-4dcf825b80fc.png)

**Comment by TheByKotik** [comment]
If I understood the request correctly, then this feature has already been implemented.
You just have to enable SteamID64.
![image](https://user-images.githubusercontent.com/40791562/102008120-01189b80-3d50-11eb-957e-744788fcc2ce.png)
![image](https://user-images.githubusercontent.com/40791562/102008138-31603a00-3d50-11eb-8202-669d9fa8ca57.png)

**Comment by crashzk** [comment]
Perfect, I will be testing, in any case it would be possible to implement in all options of this `SteamID Version?`

**Comment by TheByKotik** [comment]
> Perfect, I will be testing, in any case it would be possible to implement in all options of this `SteamID Version?`

This works in any value include zero (zero value without this flag of course).

**Comment by crashzk** [comment]
> This works in any value include zero (zero value without this flag of course).

Strange, for me it does not show option to click to open the steam in Discord Embed, this is configured like this:
```
"Discord"
{
	"Settings" // You can sum values like SteamID3 + SteamID2 = 1+2 = 3.
	{
		"Reload On" 		"3"	//	0 = OnPluginStart only, 1 = OnConfigsExecuted, 2 = Before any message send, 3 = All of this.
		"SteamID Version"	"2"	//	0 = Not use, 1 = SteamID3, 2 = Outdated SteamID2, 4 = SteamID64.
		"Map"			"1"	//	0 = Don't show map name, 1 = Show map name.
		"Time"			"1"	//	0 = Don't show time, 1 = Show time, 2 = Show timestamp, 3 = Show both.
		"Server Icon URL"	""	//	Icon of this server.
	}
	"Hooks"
	{
		"Reports"
		{
			"Color"		"0xF9D942"	//	Color for embed in HEX.
			"Icon"		""		//	Icon for this type or name of another hook to redirect.
			"Webhook"	"https://discord.com/api/webhooks/XxxXX704/FxxXXX-Z0QfdxXxXXxxozxXXXwdQ"			//	Discord webhook or name of another hook to redirect. Disabled if empty.
		}
		"Bans"
		{
			"Color"		"0xFF0000"
			"Icon"		""
			"Webhook"	"https://discord.com/api/webhooks/XxxXX704/FxxXXX-Z0QfdxXxXXxxozxXXXwdQ"
		}
		"Silences"
		{
			"Color"		"0x2B00FF"
			"Icon"		""
			"Webhook"	"https://discord.com/api/webhooks/XxxXX704/FxxXXX-Z0QfdxXxXXxxozxXXXwdQ"
		}
		"Mutes"
		{
			"Color"		"0x2B00FF"
			"Icon"		""
			"Webhook"	"https://discord.com/api/webhooks/XxxXX704/FxxXXX-Z0QfdxXxXXxxozxXXXwdQ"
		}
		"Gags"
		{
			"Color"		"0x2B00FF"
			"Icon"		""
			"Webhook"	"https://discord.com/api/webhooks/XxxXX704/FxxXXX-Z0QfdxXxXXxxozxXXXwdQ"
		}
	}
}
```

**Comment by TheByKotik** [comment]
> > Это работает при любом значении, включая ноль (конечно, нулевое значение без этого флага).
> 
> Странно, для меня не отображается возможность щелкнуть, чтобы открыть Steam в Discord Embed, это настроено так:
> 
"SteamID Version"	"6" // 2+4

**Comment by crashzk** [comment]
Worked, thank you.

**Comment by crashzk** [comment]
@TheByKotik it logically worked, but does not open the steam profile of the player that was reported, banned or received a comms.
![image](https://user-images.githubusercontent.com/32937653/102018641-2e177f00-3d4d-11eb-99ef-576a243a8106.png)
In this case, you are opening the steam of the player who reported or the admin who applied the punishment, never from the target.

**Comment by crashzk** [comment]
@TheByKotik I managed to correct it by changing this line:
https://github.com/TheByKotik/sbpp_discord/blob/babe8b9fdb0a578b404fa3558eb7e173a382002c/sourcemod/scripting/sbpp_discord.sp#L161

I changed it to:
```
GetClientAuthId( iTarget, AuthId_SteamID64, szJson[iLen], 20+1 );
```

**Comment by TheByKotik** [comment]
> @TheByKotik I managed to correct it by changing this line:
> https://github.com/TheByKotik/sbpp_discord/blob/babe8b9fdb0a578b404fa3558eb7e173a382002c/sourcemod/scripting/sbpp_discord.sp#L161
> 
> I changed it to:
> 
> ```
> GetClientAuthId( iTarget, AuthId_SteamID64, szJson[iLen], 20+1 );
> ```

Fixed in edb974f.

**Comment by joeleonjr** [comment]
Hi! I'm a security researcher @TruffleSecurity. I found a valid API key leaked in a comment in this Issue. I highly recommend rotating the API key immediately (not just deleting the comment). Here's a blog post I wrote on remediating these types of vulnerabilities: https://trufflesecurity.com/blog/remediate-leaked-api-keys-with-key-rotation.<br><br>If you can't find the leaked API key, you can use the following [TruffleHog](https://github.com/trufflesecurity/trufflehog) command to automatically scan for it:<br>`trufflehog github --repo https://github.com/<your_username>/<your_reponame> --issue-comments --only-verified`<br><br>If it's not clear where/what the security issue is, please let me know. I'd be happy to help. Please note that the API key might be in the past edit of a comment.<br><br><br><br>(cc @crashzk)
